kombu/transport/redis.py

```diff
@@ -939,9 +939,9 @@ def _get_pool(self, async=False):
         return redis.ConnectionPool(**params)
 
     def _get_client(self):
-        if redis.VERSION < (2, 10, 0):
+        if redis.VERSION < (2, 10, 4):
```

Comment:
Celery uses version `>=2.10.5`, so I would suggest to use the same here, we use since `kombu` is most frequently used from `celery`.